server/models/instance.py

```diff
@@ -135,6 +135,9 @@ def deleteInstance(self, instance, user):
         ).apply_async()
         instanceTask.get(timeout=TASK_TIMEOUT)
 
+        notify_event([str(user["_id"])], "wt_instance_deleting", "instance",
```

Comment:
Why do you cast to string every time if inside `notify_event` that argument is used for `User().load()`?

Reply:
Because I cast as str to create the sets for comparison above and wrongly think python wants consistent types.